cmd/src/actions_exec_logger.go

```diff
@@ -31,35 +33,49 @@ type actionLogger struct {
 
 	highlight func(a ...interface{}) string
 
+	progress       *progress
+	progressWriter *progressWriter
```

Comment:
I don't think there's a reason for actionLogger to know about `*progressWriter`, right?

```suggestion
	out io.Writer
```